lab_1_keywords_tfidf/main.py

```diff
@@ -3,21 +3,27 @@
 Extract keywords based on frequency related metrics
 """
 from typing import Optional, Union
+import string
+import math
 
 
-def clean_and_tokenize(text: str) -> Optional[list[str]]:
-    """
-    Removes punctuation, casts to lowercase, splits into tokens
-
```

Comment:
please keep functions signatures and docstrings as they are
otherwise, a merge conflict is likely to occur

Reply:
done.